common-app-lib/src/cmr/common_app/services/search/query_validation.clj

```diff
@@ -24,9 +25,11 @@
                         result-format)]
     (when-not (get (supported-result-formats concept-type) result-format)
       (if-let [version (mt/version-of mime-type)]
```

Comment:
is version still optional, I can not tell from the tests

Reply:
Do you mean for generics? If so yes, the rest of the search tests don't pass version so they were my "control" would comments in `generics_test` be sufficient?